lets-encrypt-r3.pem

```diff
@@ -0,0 +1,30 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
This is a derived file installed by the Makefile using `$(GET) https://letsencrypt.org/certs/lets-encrypt-r3.pem` so I don't think we want it committed to the repo.

Reply:
Removed